generate_table.Rmd

```diff
@@ -0,0 +1,624 @@
+---
+title: "Generate Table from XML Files"
+author: "Quinn White"
+date: "`r Sys.Date()`"
+output:
+  rmdformats::readthedown:
+    df_print: paged
+    code_folding: hide
+---
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo = TRUE, message=FALSE, warning=FALSE)
+```
+
+```{css,echo=FALSE}
```

Comment:
I'm assuming the CSS code here is for formatting knitted HTMLS; could you add some comments to confirm?